src/mol-repr/volume/dot.ts

```diff
@@ -103,14 +110,42 @@ export function createVolumeSphereImpostor(ctx: VisualContext, volume: Volume, k
     const count = Math.ceil((xn * yn * zn) / 10);
     const builder = SpheresBuilder.create(count, Math.ceil(count / 2), spheres);
 
+    const origin = Vec3.transformMat4(Vec3(), Vec3.create(0, 0, 0), gridToCartn);
+    const cs = count >= 4 ? Vec3.create( // Getting volume cell size. Any better way of doing this?
+        Vec3.distance(origin, Vec3.transformMat4(p, Vec3.set(p, 1, 0, 0), gridToCartn)),
+        Vec3.distance(origin, Vec3.transformMat4(p, Vec3.set(p, 0, 1, 0), gridToCartn)),
+        Vec3.distance(origin, Vec3.transformMat4(p, Vec3.set(p, 0, 0, 1), gridToCartn))
+    ) : Vec3.create(0, 0, 0);
```

Comment:
Maybe add a helper function for this to `Grid` instead, since you use it multiple times?

Also, this could contain a shear component and the cell will not be orthogonal -- will your method work as well then?

Reply:
> Maybe add a helper function for this to Grid instead, since you use it multiple times?

Ok, done.

>Also, this could contain a shear component and the cell will not be orthogonal -- will your method work as well then?

@dsehnal I tried to handle this by shrinking each dimension by `sin(axisAngle)`. It seems to work as expected in the tests I have done so far.

Reply:
>Alternatively, can you use the vectors as a basis for the coordinate system instead of taking just the magnitude and assuming orthogonal space? The pertubation would then be a random vector in that space, so you would essentially have B x [rand(), rand(), rand()] as the offset?

Should be done with last commit